roles/overlayfs_verify_missing/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+# vim: set ft=ansible:
+#
+# Runs the mount command and verifies no overlayfs is present
+#
+- name: Verify overlayfs
+  shell: mount | grep -e 'overlay on .* type overlay'
```

Comment:
ostree doesn't appear in the regex if you are using ostree admin unlock --hotfix

I can make it check for ostree in the regex when it is not hotfix and not look for the ostree if it is hotfix or I can leave it as it is.  Any thoughts?